overview.md

```diff
@@ -1,8 +1,8 @@
 ## Overview ##
 
-The [Synopsys Detect](https://synopsys.atlassian.net/wiki/spaces/INTDOCS/pages/62423113/Synopsys+Detect) plugin for Azure DevOps (formerly known as Hub Detect) supports Software Composition Analysis (SCA: open source software detection). It is architected to seamlessly integrate Synopsys Detect with Azure DevOps build and release pipelines. Synopsys Detect makes it easier to set up and scan code bases using a variety of languages and package managers across different application security techniques.  
+The [Black Duck Detect](https://blackduck.atlassian.net/wiki/spaces/INTDOCS/pages/62423113/Black+Duck+Detect) plugin for Azure DevOps (formerly known as Hub Detect) supports Software Composition Analysis (SCA: open source software detection). It is architected to seamlessly integrate Black Duck Detect with Azure DevOps build and release pipelines. Black Duck Detect makes it easier to set up and scan code bases using a variety of languages and package managers across different application security techniques.  
```

Comment:
@shantyk Change this to https://documentation.blackduck.com/bundle/detect/page/integrations/azureplugin/azure.html same as other references, the other atlassian docs is not operational anymore